src/Tasks/Microsoft.NET.Build.Tasks/ReferenceInfo.cs

```diff
@@ -13,12 +14,21 @@ public class ReferenceInfo
         public string Name { get; }
         public string Version { get; }
         public string FullPath { get; }
+        public string FileName => Path.GetFileName(FullPath);
+
+        private List<ResourceAssemblyInfo> _resourceAssemblies;
+        public IEnumerable<ResourceAssemblyInfo> ResourceAssemblies
```

Comment:
Actually this doesn't work because we need to add to the list, but want to expose the member as IEnumerable.